elife/goaws.sls

```diff
@@ -0,0 +1,25 @@
+goaws-install:
+    cmd.run:
+        - name: go get github.com/p4tin/goaws
+        - env:
+            - GOPATH: /usr/local
+        - require:
+            - pkg: golang-go
+            - aws-cli
+
+goaws-init:
+
+    file.managed:
+        - name: /etc/init/goaws-init.conf
+        - source: salt://elife/config/etc-init-goaws-init.conf
+        - mode: 755
+        - template: jinja
+        - require:
+            - goaws-install
+
+/var/run/goaws:
```

Comment:
checked, `/var/run` is symlinked to `/run` that is mounted with `tmpfs` and hence cleared from RAM at each reboot; therefore this mkdir needs to be moved to the Upstart script